src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -1254,7 +1254,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 If symbol Is Nothing Then
                     ' Match the native compiler which reports ERR_XmlFeaturesNotAvailable in this case.
                     Dim useSiteError = ErrorFactory.ErrorInfo(ERRID.ERR_XmlFeaturesNotAvailable)
-                    singleResult = New SingleLookupResult(LookupResultKind.NotReferencable, binder.GetErrorSymbol(name, useSiteError), useSiteError)
+                    singleResult = New SingleLookupResult(LookupResultKind.NotReferencable, Binder.GetErrorSymbol(name, useSiteError), useSiteError)
```

Comment:
what happened here?

Reply:
just changing to match case?

Reply:
Good eye
Looks like VS ran auto-formatting after I did a search/replace that affected this file.
That said, the new capitalization is correct (this is a static method).